src/main/java/io/cryostat/net/web/http/api/v2/graph/ActiveRecordingsFetcher.java

```diff
@@ -97,9 +97,17 @@ public Active getAuthenticated(DataFetchingEnvironment environment) throws Excep
             String recordingName = filter.get(FilterInput.Key.NAME);
             recordings =
                     recordings.stream()
-                            .filter(r -> Objects.equals(r.getName(), recordingName))
+                            .filter(r -> r.getName().equals(recordingName))
```

Comment:
Any reason to change this `Objects.equals()` call?

Reply:
just following the consistency of all other classes so they behave the same

Reply:
Do the others not use `Objects.equals()`? How does `Objects.equals(r.getName(), recordingName))` behave differently from `r.getName().equals(recordingName)`?